### PR TITLE
FACES-2456 Make it easier for developers to improve the Showcase

### PR DESCRIPTION
**Pull request by stiemannkj1**

### CODE REVIEW

**Comment by ngriffin7a** [comment]
Merged, thank you.

File: demos/showcase/showcase-webapp/src/main/webapp/views/component.xhtml [hunk 1]
```diff
@@ -41,6 +41,23 @@
 					useWrapMode="false" height="800" width="100%" value="#{useCaseCodeExample.rawText}" />
 			</alloy:tab>
 		</alloy:tabView>
+		<alloy:buttonRow class="text-center">
```
Comment (ngriffin7a) [inline demos/showcase/showcase-webapp/src/main/webapp/views/component.xhtml:44]: @stiemannkj1 FYI, after downgrading to JSF 2.1 Mojarra started complaining about this line. I'll be changing the class attribute to styleClass shortly.

Comment (stiemannkj1) [inline demos/showcase/showcase-webapp/src/main/webapp/views/component.xhtml:44]: Oops, thanks for catching that. It is supposed to be `styleClass`

File: demos/showcase/showcase-webapp/src/main/webapp/views/component.xhtml [hunk 2]
```diff
@@ -41,6 +41,23 @@
...
+			<c:set var="showcaseComponentSuite" value="alloy" />
+			<c:set var="showcaseSourceContainer" value="webapp" />
+			<c:if test="#{selectedComponentPrefix == 'bridge' || selectedComponentPrefix == 'portal' || selectedComponentPrefix == 'portlet'}">
+				<c:set var="showcaseComponentSuite" value="portal" />
+				<c:set var="showcaseSourceContainer" value="portlet" />
+			</c:if>
+			<alloy:button class="btn-mini"
```
Comment (stiemannkj1) [inline demos/showcase/showcase-webapp/src/main/webapp/views/component.xhtml:51]: Seems like a problem here as well.

File: demos/showcase/showcase-webapp/src/main/webapp/views/component.xhtml [hunk 3]
```diff
@@ -41,6 +41,23 @@
...
+				onclick="window.open('https://github.com/liferay/liferay-faces-#{showcaseComponentSuite}/edit/master/demos/showcase/showcase-#{showcaseSourceContainer}/src/main/webapp/WEB-INF/component/#{selectedComponentPrefix}/#{selectedComponentLowerCaseName}/#{selectedComponentUseCaseName}/#{selectedComponentCamelCaseName}.xhtml');">
+				<alloy:icon name="github" class="showcase-btn-icon" />
+				<alloy:outputText value="#{i18n['improve-this-example']}!" />
+			</alloy:button>
+			<alloy:button class="btn-mini"
```
Comment (stiemannkj1) [inline demos/showcase/showcase-webapp/src/main/webapp/views/component.xhtml:56]: And here.

Comment (ngriffin7a) [inline demos/showcase/showcase-webapp/src/main/webapp/views/component.xhtml:56]: Yep I fixed them all.

Comment (stiemannkj1) [inline demos/showcase/showcase-webapp/src/main/webapp/views/component.xhtml:56]: Thanks

**Pull request closed without merging**